src/eei.cpp

```diff
@@ -818,9 +818,9 @@ bool exceedsUint128(evmc_uint256be const& value) noexcept
     takeInterfaceGas(GasSchedule::copy * ((int64_t(length) + 31) / 32));
   }
 
-  bool EthereumInterface::enoughSenderBalanceFor(evmc_uint256be const& value) const
+  bool EthereumInterface::enoughSenderBalanceFor(evmc_uint256be const& value)
```

Comment:
I wonder if we can make assumptions that the host interface will only read and not make changes in certain cases. Perhaps this is a "bug" in Hera.

Reply:
I have not considered this before. In practice we can add `const` to every method because it only wraps a pointer to context. Not sure this is what we want. It makes the interface rigid.